### PR TITLE
Add a file opener

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: example/src/index.ts [hunk 1]
```diff
@@ -54,14 +55,7 @@ function main(): void {
   dock.spacing = 8;
 
   fbWidget.openRequested.connect((fb, path) => {
```
Comment (blink1073) [inline example/src/index.ts:57]: @jasongrout, the notebook opener can just hook up to `openRequested` and use that, we can just filter on path.

Comment (blink1073) [inline example/src/index.ts:57]: `openRequested` was introduced in `0.2.10` of this library.

**Comment by blink1073** [comment]
I'm putting this here because it will be growing in scope and might be reusable outside of a plugin.

**Comment by afshin** [comment]
Looks good to me.

**Comment by dwillmer** [comment]
:+1:

**Pull request merged**